examples/gadgets/withfilter/trace/network/network.go

```diff
@@ -0,0 +1,176 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/cilium/ebpf/rlimit"
+	containercollection "github.com/inspektor-gadget/inspektor-gadget/pkg/container-collection"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/container-collection/networktracer"
+	containerutils "github.com/inspektor-gadget/inspektor-gadget/pkg/container-utils/types"
+	tracernetwork "github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/trace/network/tracer"
+	tracernetworktypes "github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/trace/network/types"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/operators"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/operators/kubeipresolver"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/operators/kubenameresolver"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/socketenricher"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/types"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/utils/host"
```

Comment:
nit: we organized the the imports in 3 groups:
- standard 
- external 
- internal

Reply:
Is there any script to do that that I can use? I run the source code through the goimports tool.

Reply:
`goimports -w -local github.com/inspektor-gadget/inspektor-gadget examples/gadgets/withfilter/trace/network/network.go` should work.

Reply:
Hopefully it looks better now after running:

```bash
goimports -w -local github.com/inspektor-gadget/inspektor-gadget network.go 
```